data-prepper-plugins/geoip-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/GeoDataField.java

```diff
@@ -0,0 +1,26 @@
+package com.amazon.dataprepper.plugins.prepper.geoip.provider;
+
+public enum GeoDataField {
+    IP("ip"),
+    CITY_NAME("geo.city_name"),
+    COUNTRY_NAME("geo.country_name"),
```

Comment:
Are we using the enum values anymore? ("geo.county_name" is declared in LocationData too).

Could even use these enums directly in LocationData, and change the class name to LocationDataField.